src/common/form/FormField.js

```diff
@@ -18,6 +18,7 @@ type BaseProps = {
   blurOnSubmit?: boolean,
   nextFocusTo?: string,
   onSubmitEditing?: Function,
+  parseOnSubmit?: (value: any) => any,
```

Comment:
Can this be typed more strongly? `<TObject>(value: TObject) => $Values<TObject>`

---

I don't fully understand what this function is for so maybe that won't work.

Reply:
Yeah..I'll make generics for form/FormStore/FormField etc in the future...I just have hard time to make it all well-typed. This function is like daoApi.translator.toApi, but on the app FormField level.
I need this because doing all those kind of translations inside DAO translations are very limited and makes different pickers very hard to use.